web/src/Root.jsx

```diff
@@ -136,6 +137,27 @@ class Root extends PureComponent {
     this.setState({ initSessionId: "" });
   };
 
+  checkIsHelmManaged = async () => {
+    try {
+      const res = await fetch(`${process.env.API_ENDPOINT}/is-helm-managed`, {
+        headers: {
+          Authorization: Utilities.getToken(),
+          "Content-Type": "application/json",
+        },
+        method: "GET",
+      });
+      if (res.ok && res.status == 200) {
```

Comment:
a nit and you don't have to update but would recommend using `===` in JS because it helps with testing for strict equality